exporter/redis.go

```diff
@@ -502,13 +489,26 @@ func extractConfigMetrics(config []string, addr string, alias string, scrapes ch
 		}
 
 		if val, err := strconv.ParseFloat(strVal, 64); err == nil {
-			scrapes <- scrapeResult{Name: fmt.Sprintf("config_%s", config[pos*2]), Addr: addr, Alias: alias, Value: val}
+			e.registerGaugeValue(ch, fmt.Sprintf("config_%s", config[pos*2]), val)
 		}
 	}
 	return
 }
 
-func (e *Exporter) extractTile38Metrics(info []string, addr string, alias string, scrapes chan<- scrapeResult) error {
+func (e *Exporter) registerGaugeValue(ch chan<- prometheus.Metric, metric string, val float64, labels ...string) {
```

Comment:
The name of this function doesn't make it obvious that this is actually a ConstMetric.

Reply:
Changed.